src/modules/ModuleLiquidity/store/remove.ts

```diff
@@ -76,6 +76,10 @@ function usePrepareSupply(props: {
     usePromiseLog(supplyState, 'liquidity-remove-supply')
     useNotifyOnError(supplyState, notify, 'Supply failed')
 
+    wheneverFulfilled(supplyState, () => {
+      useLiquidityListStore().quickPoll = true
```

Comment:
I am not sure if "using" a store in a watcher callback is a good idea. Hopefully, it works, but in general any composables should be called synchronously during setup.

Reply:
Moved outside